### PR TITLE
Feature/horroy

### PR DESCRIPTION
**Pull request by danieltolmon**
TESTING
Array.from()
Array.isArray() --> Horroy.isHorroy()
Array.of()
Array.prototype.fill()
Array.prototype.filter()
Array.prototype.find()
Array.prototype.forEach()
Array.prototype.indexOf()
Array.prototype.join()
Array.prototype.map()
Array.prototype.pop()
Array.prototype.push()
Array.prototype.reduce()
Array.prototype.reverse()
Array.prototype.shift()
Array.prototype.slice()
Array.prototype.some()
Array.prototype.splice()
Array.prototype.toString()
Array.prototype.unshift()

SPECS
Array.prototype.concat()
Array.prototype.copyWithin()
Array.prototype.every()
Array.prototype.findIndex()
Array.prototype.flat()
Array.prototype.flatMap()
Array.prototype.includes()
Array.prototype.lastIndexOf()
Array.prototype.reduceRight()
Array.prototype.sort(): Array.prototype.toSource():

### CODE REVIEW

File: staff/daniel-toledo/horroy/scripts/horroy.spec.js [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    });
+    it('sould concatenate more than one input', function(){
+    
+        res=h.concat(5,6,7)
+
+        expect(res.toString()).toBe([1,2,3,4,5,6,7].toString())
+    });
+    it('sould concatenate more than one input with Horroy in the middle', function(){
+        var h2=new Horroy('this is a', 'Horroy');
+        res=h.concat(5,6,h2,7)
```
Comment (Arcoders) [inline staff/daniel-toledo/horroy/scripts/horroy.spec.js]: you should declare the variables

Comment (danieltolmon) [reply staff/daniel-toledo/horroy/scripts/horroy.spec.js]: uff there was a lot!! I was running to much.
Thanks for the feedback ;)

File: staff/daniel-toledo/community.html [hunk 2]
```diff
@@ -1,249 +0,0 @@
-<!DOCTYPE html>
```
Comment (maryammdot) [inline staff/daniel-toledo/community.html:1]: Do you have that file empty? Or you deleted the file?

Comment (danieltolmon) [reply staff/daniel-toledo/community.html:1]: I have delated... but it is following me :(

**Pull request merged**